src/typescript/decorators.d.ts

```diff
@@ -0,0 +1,54 @@
+/// <reference path="../../node_modules/reflect-metadata/Reflect.d.ts" />
```

Comment:
Is [this](https://github.com/rbuckton/ReflectDecorators) the spec defining `Reflect.getMetadata`?